### PR TITLE
Update react to 18

### PR DESCRIPTION
**Pull request by azhurauski-sc**
Just update to never version of react to use latest version of chakra-ui.

### CODE REVIEW

**Comment by azhurauski-sc** [comment]
@skylarrichard123 when you have time please have a quick look maybe you will notice something works\looks wrong.

**Comment by skylarrichard123** [comment]
What's the context here?

**Comment by azhurauski-sc** [comment]
> What's the context here?

To use new features and make some things easier to use. No major changes everything should look the same way.

File: src/components/LightMode.tsx [hunk 1]
```diff
@@ -0,0 +1,6 @@
+import { FC, ReactNode } from 'react'
+import { LightMode as OrigLightMode } from '@ironfish/ui-kit'
+
+const LightMode: FC<{ children: ReactNode }> = OrigLightMode
+
+export default LightMode
```
Comment (ndemidovich-sc) [inline src/components/LightMode.tsx:6]: Need move it into ui-kit.

**Review by ndemidovich-sc — CHANGES_REQUESTED** [review]

**Comment by skylarrichard123** [comment]
Is this branch up to date with all our recent changes?

File: src/components/LightMode.tsx (hunk 1 above)
Comment (azhurauski-sc) [reply src/components/LightMode.tsx]: Seems that I have some node_module react types conflict, fixed.

**Review by ndemidovich-sc — APPROVED** [review]

**Comment by skylarrichard123** [comment]
Screen Shot 2022-12-14 at 9 49 51 AM

There should be a space between 0.00 and $IRON

**Review by skylarrichard123 — CHANGES_REQUESTED** [review]
Just the comment about the space in the send field, but not sure if that's got anything to do with react 18

**Review by skylarrichard123 — APPROVED** [review]

**Pull request merged**